src/components/Books/Books.module.css

```diff
@@ -1,24 +1,150 @@
 .main_page {
   height: 100vh;
-  background-color: #fafafa;
+  padding: 0 6.25rem;
 }
 
 .book_section_container {
-  padding-top: 20px;
+  margin: 7.5rem 0 0;
+  padding: 0;
 }
 
```

Comment:
- [ ] It would be better if you add the progress section that is on the original design. This section does not have to work. You can use an image for this section to follow the design. 

![tempsnip](https://user-images.githubusercontent.com/64173189/150396461-390dec6a-3e05-44a8-963d-994972599128.png)

Kindly add this. 😄 🛠️

Reply:
@BrianSammit 

Thanks for your review.

If I include the progress section, it will be the same for all added books, which is why I decided to leave it out. We also don't have to implement it, according to the project description. Could you just double-check that? Thank you!

Reply:
@Mre55 

Yes, I know there is gonna be the same image but there is no problem with that. You are right you don't have to implement the progress section, that is why I ask you to just put an image for this part of the design. For this milestone, you should match Zeplin's design so you are missing this part. 

best regards.

Reply:
Added.